### ISSUE TITLE
[] UV should display in the order of source identifier, despite the order it's listed in the CSV

### ISSUE THREAD

**Issue by orangewolf** [scientist-softserv/louisville-hyku#52]

**Comment by orangewolf** [comment]
Related merge requests
---

[UV child level metadata and customizations](https://gitlab.com/notch8/louisville-hyku/-/merge_requests/151)

**Comment by orangewolf** [comment]
Shana Septemer 2022
---

We weren't going to work on this due to scope and time, however this issue was resolved while @kirkkwang was working on #124 This can be tested once #124 is merged into main/deployed to staging. 

This passes QA: 
We tested this on staging by making a new importer of a csv that was intentionally out of order. 

Before, even though we applied custom changes to order them by identifier on the parent work's show page, the UV still rendered the images in the way that the CSV listed it. 

Now, even though we imported a incorrectly ordered CSV, the images are displayed correctly (sorted by identifier)

Demo importer: https://lv-hyku-staging.notch8.cloud/importers/69?locale=en

Demo parent work: https://lv-hyku-staging.notch8.cloud/concern/images/3807babe-013a-44c8-bc4a-cb5633167094?locale=en


![Image](https://user-images.githubusercontent.com/1054448/203322427-459f717b-e114-48e2-bb6c-a53ed7d5fe67.png)


Sample file: https://drive.google.com/file/d/18G40HMAcTy1w4X2ZoA1Ptay7vhp4k0Jd/view?usp=sharing


## Testing Instructions

This ticket is dependent upon #124 

Create a CSV that has a parent and child association. They should all have source identifiers. All of the children should have images. 

Intentionally unorder the child rows so that the source identifier is mixed up and not in alphanumeric order. Zip it with the files directory of images.

Use bulkrax CSV parser to import it. 

Even though the source identifiers are out of order, the images should be displayed in the UV and items section, in alphanumeric order.

## Note
For testing, you can't re use the sample file above without changing the source identifiers.
